safe-global/web-core
# Fix: sort apps alphabetically

## What it solves

Resolves #1669

## How this PR fixes it

The apps are sorted on the client-side now (because they come unsorted from the backend).

I've also optimized the search a bit.

## How to test it
Load the safe apps page and make sure it's alphabetically ordered.
Filter by category, it should be still sorted.

## Analytics changes
None.
